index.html

```diff
@@ -296,141 +292,176 @@
     var daySpan = document.getElementById("day");
     daySpan.firstChild.textContent = dateToString(dateObj);
   }
-  function renderSchedule(dateObj, schedule, type, scheduleElement) {
-    var school;
-    if (schedule.grade <= 8) {
-      school = "MS";
+  function getFunnyImageObj(dayEventObj) {
```

Comment:
This function would probably be better if it just took the event object and returned the URL - suggest calling it getRandomImage.